api/src/main/java/pl/kolendateam/dadcard/characterCard/CharacterSkillController.java

```diff
@@ -0,0 +1,63 @@
+package pl.kolendateam.dadcard.characterCard;
+
+import java.util.HashMap;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import pl.kolendateam.dadcard.characterCard.dto.CharacterSkillDTO;
+import pl.kolendateam.dadcard.characterCard.entity.CharacterClass;
+import pl.kolendateam.dadcard.characterCard.entity.CharacterSkill;
+import pl.kolendateam.dadcard.characterCard.repository.CharacterClassRepository;
+import pl.kolendateam.dadcard.characterCard.repository.CharacterSkillRepository;
+
+@RestController
+@RequestMapping("character")
+public class CharacterSkillController {
+    
+
+    CharacterSkillRepository characterRaceRepository;
+    CharacterClassRepository characterClassRepository;
+
+    @Autowired
+    public CharacterSkillController(CharacterSkillRepository characterRaceRepository, CharacterClassRepository characterClassRepository){
+        this.characterRaceRepository = characterRaceRepository;
+        this.characterClassRepository = characterClassRepository;
+    }
+
+    @PostMapping(value="chooseskill",consumes = {"application/json"})
+    public CharacterSkill createCharacterSkill(@RequestBody CharacterSkillDTO characterSkillDTO){
```

Comment:
Przyczepie sie ale tarajmy sie nie zwracac encji tylko DTO, bo to potencjalnie niebezpieczne. Bo gdy z rozwojem aplikacji bedziesz mial w encji jakies tajne pola to beda w tym requescie szly poza aplikacje.